pkg/workload/resource_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package workload
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	corev1 "k8s.io/api/core/v1"
+	nodev1 "k8s.io/api/node/v1"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/pkg/controller/core/indexer"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+func TestAdjustResources(t *testing.T) {
```

Comment:
Also add cases as:
- runtime exists, but pod doesn't point to it.
- runtime exists, but runtime's podOverhead doesn't exist
- runtime exists, podOverhead also exists

Reply:
Thanks,now I have increased more tests to cover as much cases as possible.